module/default.nix

```diff
@@ -95,6 +95,10 @@ let
         description = "Symbolically link decrypted file to absolute paths";
       };
     };
+
+    config = {
+      path = mkDefault name;
```

Comment:
Is there a reason the path default is here rather than in the path option on line 65?

Reply:
I borrowed that coding style from nixpkgs. It helps to make clear on first sight, where the `name` is used.